spec.html

```diff
@@ -22592,6 +22603,15 @@ <h1>InitializeEnvironment ( ) Concrete Method</h1>
                 1. Let _resolution_ be ? _importedModule_.ResolveExport(_in_.[[ImportName]]).
                 1. If _resolution_ is *null* or `"ambiguous"`, throw a *SyntaxError* exception.
                 1. Call _envRec_.CreateImportBinding(_in_.[[LocalName]], _resolution_.[[Module]], _resolution_.[[BindingName]]).
+            1. Let _moduleCxt_ be a new ECMAScript code execution context.
+            1. Set the Function of _moduleCxt_ to *null*.
+            1. Assert: _module_.[[Realm]] is not *undefined*.
+            1. Set the Realm of _moduleCxt_ to _module_.[[Realm]].
+            1. Set the ScriptOrModule of _moduleCxt_ to _module_.
+            1. Set the VariableEnvironment of _moduleCxt_ to _module_.[[Environment]].
+            1. Set the LexicalEnvironment of _moduleCxt_ to _module_.[[Environment]].
+            1. Set _module_.[[Context]] to _moduleCxt_.
+            1. Push _moduleCxt_ onto the execution context stack.
```

Comment:
```suggestion
            1. Push _moduleCxt_ onto the execution context stack; _moduleCxt_ is now the running execution context.
```
(i.e., the usual wording)